query-graphs/src/tree-rendering.ts

```diff
@@ -765,7 +765,7 @@ export function drawQueryTree(target: HTMLElement, treeData: TreeDescription) {
         nodeUpdate
             .filter(function(d) {
                 return (
-                    Object.getOwnPropertyNames(getDirectProperties(d)).length || // eslint-disable-line indent
+                    Object.getOwnPropertyNames(getDirectProperties(d.data)).length || // eslint-disable-line indent
```

Comment:
I don't recall, does "d" always have a "data" to check for length?

Reply:
The `data` member exists on each node, yes.
`d` itself is the node provided by the d3-hierarchy library. It contains all the visual layout information (e.g., x and y positions, width, height), while `d.data` contains the "logical" node (i.e.: property maps etc., symbol names, ...) as returned by our format loaders